src/main/resources/operatorhub/mta-operator/0.0.1/manifests/windup-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,172 @@
+    mediatype: image/png
+  links:
+  - name: Website
+    url: https://developers.redhat.com/products/mta
+  - name: GitHub
+    url: https://github.com/windup/windup-operator
+  customresourcedefinitions:
+    owned:
+    - description: Represents an MTA application
+      displayName: MTA application
+      kind: Windup
+      name: windups.windup.jboss.org
+      version: v1
+  displayName: MTA Operator
```

Comment:
@apinnick can we use `MTA` acronym? or should we use `Migration Toolkit for Applications`?
I can see MTC is using `Migration Toolkit for Containers` as operator name.

Reply:
We should use the full name: `Migration Toolkit for Applications`.

Reply:
Thanks @apinnick 
```suggestion
  displayName: Migration Toolkit for Applications Operator
```

Reply:
:+1:

Reply:
Should I replace MTA for "Migration Toolkit for Applications" everywhere ? @mrizzi @apinnick
I mean : 
* customresourcedefinition.description : Represents an MTA application
* customresourcedefinition.displayname : MTA application
* support: MTA
* spec.maintainers.name: MTA

Reply:
My understanding from what previous docs guy told was that "titles", like the operator name in the operators catalog, must not use acronym because the user has no way to understand what does `MTA` means.
Then, once you get into descriptions, you can introduce the acronym and hence use it in the doc.
This is what I know.
Then I fully rely on @apinnick to confirm or contradict this

Reply:
Confirming. We should use "Migration Toolkit for Applications" everywhere for the operator.

Reply:
Ok, then I will replace MTA for "Migration Toolkit for Applications" here too : 

    customresourcedefinition.description : Represents an MTA application
    customresourcedefinition.displayname : MTA application
    support: MTA
    spec.maintainers.name: MTA